components/atom/checkbox/src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomCheckbox'
+
+const ERROR_STATES = {
+  ERROR: 'error',
+  SUCCESS: 'success'
+}
+
+const getErrorStateClass = errorState => {
+  if (errorState) return `${BASE_CLASS}--${ERROR_STATES.ERROR}`
+  if (errorState === false) return `${BASE_CLASS}--${ERROR_STATES.SUCCESS}`
+  return ''
+}
+
+const AtomCheckbox = ({
+  id,
+  disabled,
+  checked,
+  onChange: onChangeFromProps,
+  errorState,
+  ...props
+}) => {
+  const handleChange = ev => {
+    const {checked, name} = ev.target
+    if (!disabled) onChangeFromProps(ev, {name, value: checked})
+  }
+
+  const className = cx(BASE_CLASS, getErrorStateClass(errorState))
+
+  return (
+    <Fragment>
```

Comment:
Should it has shorthand syntax in order to be compliant with latest sui lint update?